### PR TITLE
Issue #68 - Added SSL Support

### PR DESCRIPTION
**Pull request by FireDrunk**
Added SSL Support in ERB Template.

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -152,7 +152,8 @@
   $refresh              = $uchiwa::params::refresh,
   $sensu_api_endpoints  = $uchiwa::params::sensu_api_endpoints,
   $users                = $uchiwa::params::users,
-  $auth                 = $uchiwa::params::auth
+  $auth                 = $uchiwa::params::auth,
+  $ssl                  = $uchiwa::params::ssl,
```
Comment (solarkennedy) [inline manifests/init.pp]: Can you add a docstring to explain how to use this? Bonus points for a readme update

File: templates/etc/sensu/uchiwa.json.erb [hunk 2]
```diff
@@ -47,6 +47,12 @@
     "auth": {
       "publickey": "<%= @auth['publickey']  %>",
       "privatekey": "<%= @auth['privatekey']  %>"
+    }<%= ',' if @ssl.size == 2 %>
```
Comment (solarkennedy) [inline templates/etc/sensu/uchiwa.json.erb]: It would be easier to just slide this into the `if` block below no?

File: manifests/init.pp (hunk 1 above)
Comment (FireDrunk) [inline manifests/init.pp]: Done in commit: 108c3b3

File: templates/etc/sensu/uchiwa.json.erb (hunk 2 above)
Comment (FireDrunk) [inline templates/etc/sensu/uchiwa.json.erb]: I don't think thats possible. The ',' should come after the '}' according to JSON standards.
I just copied the same style from the auth section.

**Comment by solarkennedy** [comment]
Last thing to do is check out the tests, which look like the same ones that are broken in master :( 

So I think this is good to go. Thanks!

**Pull request merged**